src/styles/calendar.scss

```diff
@@ -0,0 +1,68 @@
+@import "./utils/variables.scss";
+@import "./utils/mixins.scss";
```

Comment:
shouldn't be there

Reply:
@silchyn, If i write this:
      @import "./utils/variables.scss";
      @import "./utils/mixins.scss";
in main.scss it dosen't work.

If I am not mistaken, @Anna-Tkachenko said, that sometimes it's ok, if we connetc variables and mixins in block page where this variables and mixins will be used.

Please accept this.